src/ios/CDVWKInAppBrowser.m

```diff
@@ -1175,6 +1175,18 @@ - (UIColor *)colorFromHexString:(NSString *)hexString {
 
 #pragma mark WKNavigationDelegate
 
+- (void)webView:(WKWebView *)theWebView didReceiveAuthenticationChallenge:(nonnull NSURLAuthenticationChallenge *)challenge completionHandler:(nonnull void (^)(NSURLSessionAuthChallengeDisposition, NSURLCredential * _Nullable))completionHandler {
+    NSString *domain = theWebView.URL.absoluteURL.host;
```

Comment:
I guess `domain` could also be consistently named `host` here as well.  (also slightly more correct, since the host isn't always a domain name, but can also be an host IP.

Reply:
You're right, a `host` can be more than just domain. Thanks for pointing it out 👍